throw an error if prerenderable routes are not prerendered (#6974)
sveltejs/kit: documentation/docs/13-page-options.md

Note that this will disable client-side routing for any navigation from this page, regardless of whether the router is already active.

#### Troubleshooting

> If you encounter an error like 'Route ... is marked as prerenderable, but was not prerendered' it's because the route in question (or a parent layout, if it's a page) has `export const prerender = true` but the page wasn't actually prerendered.

Reviewer: Out of scope for this PR, but I think a dedicated page that lists all errors thrown by Kit would be nice. The errors thrown by Kit would then also have a link pointing to that site where you can read more about that error.

> If you encounter an error like 'Route ... is marked as prerenderable, but was not prerendered' it's because the route in question (or a parent layout, if it's a page) has `export const prerender = true` but the page wasn't actually prerendered.

Reviewer: ```suggestion
If you encounter an error like 'The following routes were marked as prerenderable, but were not prerendered' it's because the route in question (or a parent layout, if it's a page) has `export const prerender = true` but the page wasn't actually prerendered.
```
